DVTShowcase/AboutViewController.swift

```diff
@@ -33,33 +34,33 @@ class AboutViewController: DVTShowcaseViewController, UITableViewDelegate, UITab
     }
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCell(withIdentifier: "CellIdentifier", for: indexPath) as! AboutTableViewCell
+        let cell = tableView.dequeueReusableCell(withIdentifier: "AboutCellIdentifier", for: indexPath) as! AboutTableViewCell
         cell.layer.cornerRadius = 2.0
         cell.layer.borderWidth = 0.5
         cell.layer.borderColor = UIColor.lightGray.cgColor
         cell.selectionStyle = UITableViewCellSelectionStyle.none
 
         cell.aboutLabel.text = self.aboutDVT
-//        cell.emailButton.addTarget(self, action: #selector(emailButtonPressed(sender:)), for: .touchUpInside)
-//        cell.emailIconButton.addTarget(self, action: #selector(emailButtonPressed(sender:)), for: .touchUpInside)
-//        cell.emailButton.tag = indexPath.section
-//        cell.emailIconButton.tag = indexPath.section
-//        
-//        cell.callButton.addTarget(self, action: #selector(callButtonPressed(sender:)), for: .touchUpInside)
-//        cell.callIconButton.addTarget(self, action: #selector(callButtonPressed(sender:)), for: .touchUpInside)
-//        cell.callButton.tag = indexPath.section
-//        cell.callIconButton.tag = indexPath.section
-//        
-//        cell.navigateButton.addTarget(self, action: #selector(navigateButtonPressed(sender:)), for: .touchUpInside)
-//        cell.navigateIconButton.addTarget(self, action: #selector(navigateButtonPressed(sender:)), for: .touchUpInside)
-//        cell.navigateButton.tag = indexPath.section
-//        cell.navigateIconButton.tag = indexPath.section
+        cell.websiteButton.addTarget(self, action: #selector(openWebsitePressed(sender:)), for: .touchUpInside)
```

Comment:
Given that you are using custom cell types here, why don't you embed the logic for configuring the cell in the cell itself. By configuring the cell's presentation details in this method you are exposing what is essentially internal information about the cell and violating encapsulation. Rather create a method on the cell that takes the relevant arguments and does all the configuration.